### DISCUSSION TITLE
[3.x]: Runtime cache stores too much information

### DISCUSSION THREAD

**Discussion by palomamtnez** [craftcms/cms#10837] [Q&A] [answered]
### Description

Is it possible to disable the cache from the `/container/application/storage/runtime/cache` directory?
Or what would be the best approach to get fewer files in here?

I'm saying this because I have two sites; one in staging and one in production.
Between the two containers, there are 8 million files in these directories alone and over 30G of disk space. 
To put this into perspective, the container that is in production has about 23G of files on the server, and about 20G of that is in this cache directory alone.

These containers have been running for about three years, so is it like something that periodically we need to be cleaning?

### Craft CMS version

 3.7.11

### PHP version

7.4.27

### Operating system and version

Linux 5.4.0-51-generic

### Database type and version

MySQL 5.7.32

### Image driver and version

Imagick 3.7.0 (ImageMagick 6.9.10-23)

### Installed plugins and versions

![Screen Shot 2022-03-29 at 5 06 45 PM](https://user-images.githubusercontent.com/13962572/160531182-453c1473-019c-4697-8570-809c7bf971c5.png)

**Comment by brandonkelly** [comment, marked as answer]
You can configure Craft to store its data caches elsewhere, such as the database or Redis, by [overriding the `cache` component](https://craftcms.com/docs/3.x/config/#cache-component).

> These containers have been running for about three years, so is it like something that periodically we need to be cleaning?

Yes, it’s a good idea to clear out your caches from time to time, particularly after updates, where the cache keys may change and old caches may no longer be needed.
